<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPreferencesFolder</class>
 <widget class="QDialog" name="DialogPreferencesFolder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Folder preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblFolder">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Folder:</string>
     </property>
     <property name="buddy">
      <cstring>editFolder</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="editFolder"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="btnChooseFolder">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="keepOrganized">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Keep this folder organized.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupFormat">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="10" column="0">
       <widget class="QLabel" name="lblOrgFormat">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Filename format:</string>
        </property>
        <property name="buddy">
         <cstring>editOrgFormat</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="3">
       <widget class="QLineEdit" name="editOrgFormat">
        <property name="maxLength">
         <number>256</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblFormatAlbum">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Album title:</string>
        </property>
        <property name="buddy">
         <cstring>editOrgAlbumDefault</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editOrgAlbumDefault">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="editOrgYearDefault">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="editOrgAlbumEmpty">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="editOrgTrackEmpty">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblFormatTrack">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Track number:</string>
        </property>
        <property name="buddy">
         <cstring>editOrgTrackDefault</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblFormatYear">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Year:</string>
        </property>
        <property name="buddy">
         <cstring>editOrgYearDefault</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="editOrgYearEmpty">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblFormatDisc">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Disc number:</string>
        </property>
        <property name="buddy">
         <cstring>editOrgDiscDefault</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="editOrgArtistEmpty">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editOrgArtistDefault">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblFormatTitle">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Title:</string>
        </property>
        <property name="buddy">
         <cstring>editOrgTitleDefault</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblFormatArtist">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Artist name:</string>
        </property>
        <property name="buddy">
         <cstring>editOrgArtistDefault</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblFormatDefault">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Default value</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="editOrgDiscEmpty">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="editOrgDiscDefault">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="editOrgTrackDefault">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblFormatEmpty">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Empty field</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editOrgTitleDefault">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="editOrgTitleEmpty">
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="lblOrgTitleNum">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">(%1)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="lblOrgArtistNum">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">(%2)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="lblOrgAlbumNum">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">(%3)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="lblOrgYearNum">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">(%4)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="lblOrgTrackNum">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">(%5)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="lblOrgDiscNum">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">(%6)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblFormatGenre">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Genre:</string>
        </property>
        <property name="buddy">
         <cstring>editOrgGenreDefault</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="editOrgGenreDefault"/>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="editOrgGenreEmpty"/>
      </item>
      <item row="7" column="3">
       <widget class="QLabel" name="lblOrgGenreNum">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">(%7)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lblFormatCompilation">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Compilation:</string>
        </property>
        <property name="buddy">
         <cstring>editOrgCompilation</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="editOrgCompilation"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editFolder</tabstop>
  <tabstop>btnChooseFolder</tabstop>
  <tabstop>keepOrganized</tabstop>
  <tabstop>editOrgTitleDefault</tabstop>
  <tabstop>editOrgTitleEmpty</tabstop>
  <tabstop>editOrgArtistDefault</tabstop>
  <tabstop>editOrgArtistEmpty</tabstop>
  <tabstop>editOrgAlbumDefault</tabstop>
  <tabstop>editOrgAlbumEmpty</tabstop>
  <tabstop>editOrgYearDefault</tabstop>
  <tabstop>editOrgYearEmpty</tabstop>
  <tabstop>editOrgTrackDefault</tabstop>
  <tabstop>editOrgTrackEmpty</tabstop>
  <tabstop>editOrgDiscDefault</tabstop>
  <tabstop>editOrgDiscEmpty</tabstop>
  <tabstop>editOrgGenreDefault</tabstop>
  <tabstop>editOrgGenreEmpty</tabstop>
  <tabstop>editOrgCompilation</tabstop>
  <tabstop>editOrgFormat</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
